x/oracle/keeper/keeper.go

```diff
@@ -370,13 +371,21 @@ func (k Keeper) SetPriceSnapshot(ctx sdk.Context, snapshot types.PriceSnapshot)
 
 func (k Keeper) AddPriceSnapshot(ctx sdk.Context, snapshot types.PriceSnapshot) {
 	params := k.GetParams(ctx)
-	lookbackDuration := params.LookbackDuration
+
+	// Sanity check to make sure LookbackDuration can be converted to int64
+	// Lookback duration should never get this large
+	if params.LookbackDuration > uint64(math.MaxInt64) {
```

Comment:
is Uint(MaxInt64) and MaxInt64 semantically the same value? idr the way the representation of sign works so just wanted to make sure it would be fine and not like a max Uint64 value

Reply:
Yes its semantically the same. I also verified this.